lib/cog_api/fake/server.ex

```diff
@@ -2,7 +2,8 @@ defmodule CogApi.Fake.Server do
   defstruct [
     groups: [],
     roles: [],
-    permissions: []
+    permissions: [],
+    bundles: [],
```

Comment:
maybe we should sort this alphabetically? might be nice as this list gets longer